.gitignore

```diff
@@ -104,3 +104,6 @@ cataclysm.a
 
 # coredumps
 *.core
+
+# sublime-text workspace
+*.sublime-workspace
```

Comment:
The workspaces is the user specific settings.
On Jan 24, 2016 12:09 AM, "Malkeus" <notifications@github.com> wrote:

> In .gitignore
> <https://github.com/CleverRaven/Cataclysm-DDA/pull/14963#discussion_r50630767>
> :
>
> > @@ -104,3 +104,6 @@ cataclysm.a
> >
> >  # coredumps
> >  *.core
> > +
> > +# sublime-text workspace
> > +*.sublime-workspace
>
> I don't think you'll want to add the sublime workspaces for the whole
> project to .gitignore, they will probably need to be updated in the future,
> I know the codeblocks project file gets modified frequently. If you had to
> update the workspace to make the project work, include those changes for
> review.
> Iirc several libraries were added to the codeblocks project not to long
> ago, I don't think the sublime workspace was updated at the same time.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CleverRaven/Cataclysm-DDA/pull/14963/files#r50630767>.
>